dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -1 +1,3 @@
 = render partial: 'levels/editors/fields/dsl_editors', locals: {f: f}
+= render partial: 'levels/editors/fields/bubble_choice_sublevel', locals: {f: f} if @level.is_a?(LevelGroup) || @level.is_a?(Match)
```

Comment:
I think I see what you did here -- is this just all the DSL level types not explicitly excluded in the "before" code?https://github.com/code-dot-org/code-dot-org/pull/35163/files#diff-6a11121c202ed774449d17268533b467L44